docs/UsersGuide/source/CustomizingTheWorkflow/ConfigWorkflow.rst

```diff
@@ -1295,6 +1298,9 @@ Set parameters for customizing the :term:`UPP`.
 ``TESTBED_FIELDS_FN``: (Default: "")
    The file that lists grib2 fields to be extracted for testbed files. An empty string means no need to generate testbed files.
 
+``NUMX``: (Default: 1)
+   The number of i regions in a 2D decomposition.  Each i row is distibuted to numx ranks.  Set NUMX to a value greater than 1 to enable 2D decomposition.
+
```

Comment:
```suggestion
   Note that this value will be reset to 1 automatically when using GNU compilers (i.e., ``COMPILER: gnu``) because 2D decomposition does not yet work with GNU compilers.

```